renee

```diff
@@ -1043,6 +1043,10 @@ def run(sub_args):
         sub_args, ifiles=input_files, repo_path=git_repo, output_path=sub_args.output
     )
 
+    # ensure the working dir is read/write friendly
+    workflow_path=os.path.join(sub_args.output,'workflow')
+    subprocess.call(['chmod', '-R', '777', workflow_path])
```

Comment:
A few suggestions:

1. Recommend using `os.chmod` instead of `subprocess.call` -- typically whenever there is a built-in Python way to execute system commands it should be faster to use the built-in method.
2. Let's only change the script dir to be writeable by the user, since this is only needed since the Rmarkdown files live there. Ideally we probably should've put those in a `notebooks` subdir to follow standard snakemake guidelines, but it's probably not worth the trouble at this point.
3. How about we do 755 instead of 777, this way group+others will only be able to read & execute but not write.

```suggestion
    scripts_path = os.path.join(sub_args.output,'workflow','scripts')
    os.chmod(scripts_path, 0o755)
```